.github/workflows/health-check.yml

```diff
@@ -0,0 +1,31 @@
+name: Health Check
+
+on:
+  schedule:
+    # At minute 0 past hour 0800 and 2000.
+    - cron:  '0 8,20 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10.x', '12.x', '14.x' ]
```

Comment:
Are these versions meant to be different for the health check vs tests?

Reply:
nope they should ideally be the same, i have updated them